.github/workflows/workflow.yml

```
@@ -87,14 +87,13 @@ jobs:
 
       - name: Define env (unix)
         if: ${{ runner.os != 'Windows' }}
-        run: echo "::set-env name=GITHUB_TAG::${GITHUB_REF#refs/*/}"
+        run: echo "GITHUB_TAG=$(git describe --always)" >> $GITHUB_ENV
```

Good idea, to make sure the tag is there even if not annotated, probably is better to use `--tags` (also below)
```suggestion
        run: echo "GITHUB_TAG=$(git describe --always  --tags)" >> $GITHUB_ENV
```